core/src/test/scala/integration/kafka/api/SaslPlainPlaintextConsumerTest.scala

```diff
@@ -27,9 +28,15 @@ class SaslPlainPlaintextConsumerTest extends BaseConsumerTest with SaslTestHarne
   override protected val kafkaServerSaslMechanisms = List(kafkaClientSaslMechanism)
   override protected val kafkaServerJaasEntryName =
     s"${listenerName.value.toLowerCase(Locale.ROOT)}.${JaasTestUtils.KafkaServerContextName}"
-  this.serverConfig.setProperty(KafkaConfig.ZkEnableSecureAclsProp, "true")
+  this.serverConfig.setProperty(KafkaConfig.ZkEnableSecureAclsProp, "false")
   override protected def securityProtocol = SecurityProtocol.SASL_PLAINTEXT
   override protected lazy val trustStoreFile = Some(File.createTempFile("truststore", ".jks"))
   override protected val serverSaslProperties = Some(kafkaServerSaslProperties(kafkaServerSaslMechanisms, kafkaClientSaslMechanism))
   override protected val clientSaslProperties = Some(kafkaClientSaslProperties(kafkaClientSaslMechanism))
+
+  @Before
+  override def setUp {
+    super.setUp
+    TestUtils.verifyUnsecureZkAcls(zkUtils)
```

Comment:
It's a bit odd to have this check in `setUp`. Since `setUp` is always the same, we can probably just add a single test that verifies this. If we break the code, we would have one test failure with a clear cause. Same for the other test class.